assets/extensions/examples/online-extensions.css

```diff
@@ -0,0 +1,6 @@
+/*
+  Customize the logo in the homepage header
+*/
+.navbar-pf-alt #header-logo {
```

Comment:
Any reason to not use `#header-logo` only? It's the example we have in the docs and should always be unique on the page.